### ISSUE TITLE
Simple refactoring of NivoSliderPlugin class

### ISSUE THREAD

**Issue by davidandrewpowell** [nopSolutions/nopCommerce#3800]
nopCommerce version:  4.20 beta

After fixing a separate issue in `NivoSliderPlugin`, I would suggest the following simple refactorings for this class:

i) Separate larger code sections into smaller methods
ii) Separate the large `InsertPicture` line into multiple lines for simplicity
iii) Remove redundant source code for banner picture 3
iv) Removal redundant comments for //pictures and //settings
v) Call `_webHelper.GetStoreLocation(false)` only once rather than once for each picture.
vi) Correct spelling of PLugin to Plugin in comments.

**Comment by davidandrewpowell** [comment]
See pull request [3801](https://github.com/nopSolutions/nopCommerce/pull/3801).

**Comment by davidandrewpowell** [comment]
Following feedback on this, it has been decided not to perform these refactorings at present in order to keep the plugin install methods consistent across all plugins.
